src/aics-image-viewer/components/App/index.tsx

```diff
@@ -1159,6 +1167,48 @@ export default class App extends React.Component<AppProps, AppState> {
     return { x: 0, y: 0, z: 0 };
   }
 
+  private metadataSelectors: MetadataSelectors = {
+    dimensions: (image) => {
+      const { x, y, z } = image;
+      return { Dimensions: { x: x + "px", y: y + "px", z: z + "px" } };
+    },
+    originalDimensions: (image) => {
```

Comment:
dimensions vs originalDimensions is a really interesting problem.  I think when we load multiresolution zarr data, originalDimensions are going to be wrong; they are not the "level zero" raw dimensions.